docs/current_docs/api/snippets/services/bind-services/go/main.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"context"
-	"main/internal/dagger"
+	"dagger/my-module/internal/dagger"
```

Comment:
:thinking: The `go.mod` for this still puts this as `module main` - so the previous change is actually the right one.

But also I guess this comes back to a prior discussion about the runability of these examples as modules (vs just as code snippets). So this isn't blocking.